ServerAPI/Game.cs

```diff
@@ -1,116 +1,68 @@
-﻿using System;
-using System.Linq;
-using System.Threading.Tasks;
+namespace ServerAP;
 
-namespace TicTacToe;
-public enum GameSigns
-{
-    Empty,
-    O,
-    X,
-         
-}
 public class Game
 {
     private readonly int[,] _field = new int[3, 3]
-    { // 0  1  2
+    {
+        // 0  1  2
         {7, 8, 9}, //0
         {4, 5, 6}, //1
-        {1, 2, 3}  //2
+        {1, 2, 3} //2
     };
-
+    
+    
+    public static List<Player> Players = new();
+    public TurnInfo TurnInfo { get; set; } = null!;
+    public GameState GameState { get; set; }
     public GameSigns Winner { get; set; }
     public GameSigns[,] GameField = new GameSigns[3, 3];
     public GameSigns CurrentSign = GameSigns.X;
-    public bool IsGameEnd;
-    private readonly Services _services;
-
-    public Game(Services services)
-    {
-        _services = services;
-    }
-
-    public GameSigns SignFromServer { get; set; }
-
-    public async void MakeTurn(int numpadTurnInput)
-    {
-        for (int i = 0; i < 3; i++)
-        {
-            for (int j = 0; j < 3; j++)
-            {
-                if (_field[i, j] == numpadTurnInput)
-                {
-                    await MakeTurn(i,j);
-                }
-            }
-        }
-    }
-
-    public async Task MakeTurn(int x, int y)
+    public bool IsGameEnd { get; set; }
+    public bool CanPlayerMakeTurn { get; set; }
+    
+    
+    public void MakeTurn()
     {
-        if (GameField[x, y] != GameSigns.Empty)
-            return;
-        if (!IsNowMyTurn())
-        {
-            Console.WriteLine("now it's the other player's turn");
-            await _services.WaitForTurn( this);
-            return;    
-        }
         
-        GameField[x, y] = CurrentSign;
-        await _services.ServerMakeTurn(this);
-        if (FindWinCombination(x, y))
+        GameField[TurnInfo.X, TurnInfo.Y] = CurrentSign;
```

Comment:
I think it will be easier to understand if you pass TurnInfo as an argument in that method rather then putting it in the property from outside of the class and use it here.

To me it looks more like using a global variable. You can put TurnInfo as a global variable somewhere in the code, put the value in one place, and read in another. In both cases you need to keep in mind who and when put the value inside that variable. It's hard